libr/anal/anal.c

```diff
@@ -32,9 +31,11 @@ static void meta_unset_for(void *user, int idx) {
 }
 
 static int meta_count_for(void *user, int idx) {
+	int ret;
 	RSpaces *s = (RSpaces*)user;
 	RAnal *anal = (RAnal*)s->user;
-	return r_meta_space_count_for (anal, idx);
+	ret = r_meta_space_count_for (anal, idx);
```

Comment:
no need to define, assign and return, when you can just return the result of the function here. you will save 2 lines here